src/meta_schedule/feature_extractor/utils.h

```diff
@@ -33,6 +33,8 @@ namespace tir {
 
 /*! \brief Type for multi-dimensional index */
 using MultiIndex = std::vector<PrimExpr>;
+/*! \brief Type for a region */
+using NDIntSet = Array<arith::IntSet>;
```

Comment:
NDIntSet is defined here: https://github.com/Kathryn-cat/tvm/blob/0d2153a8f7681cfa32d3f407b9ee68e9e0a9d884/src/support/nd_int_set.h#L32 (with a slightly different definition though)

Let's introduce it here via:

```C++
using tvm::support::NDIntSet;
```

Reply:
Addressed :)